dev/index.js

```diff
@@ -4,9 +4,36 @@ import Boilerplate from './Boilerplate'
 import Buoy from 'buoy'
 import VueRouter from 'vue-router'
 import router from './router'
+import VeeValidate from 'vee-validate'
+
+
+const validationConfig = {
+  errorBagName: 'errors',
+  fieldsBagName: 'fields',
+  delay: 0,
+  locale: 'en',
+  dictionary: null,
+  strict: false,
+  classes: false,
+  classNames: {
+    touched: 'touched', // the control has been blurred
+    untouched: 'untouched', // the control hasn't been blurred
+    valid: 'valid', // model is valid
+    invalid: 'invalid', // model is invalid
+    pristine: 'pristine', // control has not been interacted with
+    dirty: 'dirty' // control has been interacted with
+  },
+  events: 'blur',
+  inject: true,
+  validity: false,
+  aria: true,
```

Comment:
Is this validator truly accessible?

Reply:
It adds aria-invalid and aria-required. It doesn't associate the errors with the input because it doesn't determine how those get displayed. We still have to do that ourselves.